cdap-data-fabric-tests/src/test/java/co/cask/cdap/data/stream/NoopStreamAdmin.java

```diff
@@ -32,7 +32,7 @@
 public class NoopStreamAdmin implements StreamAdmin {
 
   @Override
-  public void dropAll() throws Exception {
+  public void dropAllInNamespace(Id.Namespace namespace) throws Exception {
```

Comment:
I was thinking about these today - Is it better to call it just ``dropAll(Id.Namespace namespace)``? since the parameter already suggests that the operation will be applied within a namespace?